wurst/boss/WeaponTypes.wurst

```diff
@@ -14,15 +15,23 @@ public interface WeaponListener
 
 public WeaponType array weapontypes
 
+function damageTarget(Entity entity, Escaper escaper, real damage)
```

Comment:
not good if we have to extend this one function for everything that should be procced by damage.
We should do this in another way, maybe via buffs or listeners